src/lib/response-triggers/unpeg.ts

```diff
@@ -125,11 +125,17 @@ export default class  Unpeg extends Trigger {
 `### HTTP Status Code 418: I'm a teapot.
 Unable to brew coffee. Or pegs.`);
 			case 6:
+				let nonAlphanumericRegex = new RegExp('[^0-9a-z]', 'gi');
```

Comment:
Should be A-z, otherwise you'll remove the uppercase characters in people's names

Reply:
Good regex uses 'A-Za-z' if case sensitive. But it doesn't matter here since it has the 'i' flag, which means it's checking case insensitively.

Reply:
https://stackoverflow.com/a/4923397/5658246

Reply:
huh, didn't know there was a case sensitivity flag.

Reply:
The options are 'gi', 'i' means case *i*nsensitive